test/cms/page_test.exs

```diff
@@ -11,6 +11,12 @@ defmodule PageTest do
   describe "from_api/1" do
     test "switches on the node type in the json response and returns the proper page struct" do
       assert %Page.Basic{} = Page.from_api(Static.basic_page_response())
+
+      diversions_data =
+        Static.basic_page_response() |> Map.put("field_page_type", [%{"name" => "Diversions"}])
```

Comment:
📓 This is just a hacky way of creating a Diversions page without creating a giant JSON blob like all other CMS test pages.